### ISSUE TITLE
Preheader not showing up on the delivered Email

### ISSUE THREAD

**Issue by priyankpardiwala** [adobe/aem-core-email-components#250]
Hello - it looks like the email core page component does not insert preheader text from page properties into the Body of the HTML. 

As a result - the delivered emails do not show the preheader text entered in Page Properties.

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES as Bug

**Comment by bpauli** [comment]
Hi @priyankpardiwala, Thank you for bringing this up!

Can you please provide an example of the expected behaviour of the preheader ? Should this text be used for the preview of an email or just as 1st element before a component can be inserted into the body? The definition of preheader is sometimes mixed with the preview text (see also https://www.litmus.com/blog/the-ultimate-guide-to-preview-text-support) Thanks!

**Comment by priyankpardiwala** [comment]
Hi @bpauli - 

Preheader would be the first element in the <body> tag. But it should have a display:none on the style attribute. 

That way even though the preheader text is not directly visible on the page, it shows up as preview of the email below the "Subject" line on the email clients. 

Here's a sample markup: 

```
<body>
	<div style="display:none;font-size:1px;color:#fefefe;line-height:1px;max-height:0px;max-width:0px;opacity:0;overflow:hidden;">
	  ${pageProperties.preHeadertext @context='html'}      
	</div>
        <!--rest of the page markup -->
</body>
```

**Comment by bpauli** [comment]
@priyankpardiwala, Can you please validate if the issue is fixed with the latest release (1.2.0) Thanks!
